### PR TITLE
feat: add frameworks titles

### PR DESCRIPTION
**Pull request by erezrokah**
Fixes https://github.com/netlify/framework-info/issues/133

### CODE REVIEW

File: src/frameworks/docusaurus-v2.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "docusaurus-v2",
+  "title": "Docusaurus 2",
```
Comment (erezrokah) [inline src/frameworks/docusaurus-v2.json:3]: Explicitly use v2 in the title as referenced in https://docusaurus.io/

File: src/frameworks/quasar-v0.17.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "quasar-v0.17",
+  "title": "Quasar",
```
Comment (erezrokah) [inline src/frameworks/quasar-v0.17.json:3]: Didn't include the version this time as it can mean `any version lower than v1`

**Review by JGAntunes — APPROVED** [review]
I take it we're already too late to change `name` for `id` without breaking a whole lot of things and requiring a whole lot of updates correct? 😅

**Comment by erezrokah** [comment]
> I take it we're already too late to change `name` for `id` without breaking a whole lot of things and requiring a whole lot of updates correct? 😅

I actually think we could do that as a breaking change.

**Pull request merged**